src/contract_client/types.ts

```diff
@@ -1,11 +1,8 @@
-import {
-  BASE_FEE,
-  Memo,
-  MemoType,
-  Operation,
-  Transaction,
-  xdr,
-} from "..";
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
if you're going to disable, a comment why is super helpful for review & posterity :pray:

Reply:
I'll add a code comment, too, but the linter wants the type-naming convention to follow PascalCase. It doesn't like that we have a bunch of types like `i32` and `XDR_BASE64` in here. Renaming them all would be a big breaking change.